src/components/Landing/QAs.tsx

```diff
@@ -130,5 +217,40 @@ export default function QAs() {
         alt: t`It'd be a lot cooler if you did`,
       },
     },
+    {
+      q: t`Who is Peel?`,
+      a: [
+        <Trans>
+          Peel is the DAO that manages Juicebox's frontend interface. You can
```

Comment:
**sugestion**: perhaps a link to the peel JB project?

Reply:
yep good idea